CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -8,22 +8,25 @@ static const double cap = 6.0;
 static const double tspar0[2] = {15.5, 12.2999};
 static const double tspar1[2] = {-3.2,-2.19142};
 static const double tspar2[2] = {32, 0};
-static const double tspar0_siPM[2] = {3., 3.}; // 3ns delay for MC and DATA, recheck later for data
+static const double tspar0_siPM[2] = {0., 0.}; // 2ns delay for MC and DATA, recheck later for data
```

Comment:
@ kpedro.
I checked the M3 scale with the additional offset of 0ns or 2ns and doens't change.
The point of this PR is to remove the energy evolution of the correction.

With M2 time I measure now 2ns as delay of the MC digi. Before the addition of the Photostatistics, it was 3 ns. Given that the M2 time change while the M2 evolve and that the HPD M2 show the two population, I prefer to not to use this extra delay and I kept only the 6ns (the delay used in the digitization).